feature_reflect_extension.go

```diff
@@ -231,8 +236,7 @@ func describeStruct(cfg *frozenConfig, typ reflect.Type) (*StructDescriptor, err
 				continue
 			}
 		}
-		tagParts := strings.Split(field.Tag.Get("json"), ",")
-		fieldNames := calcFieldNames(field.Name, tagParts[0], string(field.Tag.Get("json")))
```

Comment:
`field.Tag.Get("json")` returns a string, no need to cast it as `string` again, isn't?